### ISSUE TITLE
Smaller text fragments for question annotations

### ISSUE THREAD

**Issue by nyxpho** [etalab-ia/piaf-ml#105]
**Problem description**
Currently when we are trying to create questions to increase our dataset, we are presented with a very long text. We have agreed for the first round of annotations to just look at the first 5  - give or take - sentences and to create a question where the answer would be in that paragraph. It would be best however if the text comes already split, perhaps with some overlap between consecutive fragments so that we keep context. Next week a very junior researcher will join our team and he could help us with some additional questions. 

**Solution description**
I can do this, just let me know what is the file you are currently using as an input for the annotation platform. 

@guillim @Rob192 @psorianom @AbdenourCh

**Comment by nyxpho** [comment]
Update on this: we had an intern in the team that annotated 9 more texts, with 5 questions for each. As there were issues in the annotation platform, I could not proceed with splitting the text in smaller parts. The new squad file, containing the old questions plus the new ones is this one. The documents with new questions are: 953, 1109, 1268, 1606, 1881, 1974, 2062, 2383, 2833. Does the document look ok to you, should I make a pull request to add it? 
[squad++.zip](https://github.com/etalab-ia/piaf-ml/files/6902200/squad%2B%2B.zip)

**Comment by guillim** [comment]
Are we talking about the dataset from service-publique.fr  ?

**Comment by nyxpho** [comment]
Yes

**Comment by guillim** [comment]
Does your intern plans to make more annotations in august ? (I am thinking maybe to make one PR only when finished)

**Comment by nyxpho** [comment]
No, he will not make more annotations. Sure, we can make just one PR.
